src/tic_tac_toe/core.clj

```diff
@@ -17,5 +22,8 @@
 (defn -main
   []
   (let [utils (setup/setup-utils)]
-    (-> (setup/setup-game utils "saves" "game")
-        (play))))
+    (shutdown-listener utils)
+    (try
+      (-> (setup/setup-game utils "saves" "game")
+          (play))
+      (catch Exception e))))
```

Comment:
does this mean it's swallowing all exceptions?

Reply:
Temporarily updated to [RuntimeException](https://github.com/saralein/tic-tac-toe_clojure/pull/12/files#diff-57905e40aaa721e61a7ea7ef112d9ed3R29).  Still looking for a different solution.

Reply:
OK, removed the exception handling and replaced it with a [check](https://github.com/saralein/tic-tac-toe_clojure/pull/12/files#diff-52c69c82bcdbcc16e3156d4a9033a37cR10) to see if the user if trying to quit.  Also removed the shutdown hook since the other PR clears the screen when exiting and we won't see the message.